tests/api/v2/handlers/test_contacts_api.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from http import HTTPStatus
+
+
+@pytest.fixture
+def contact_report_data(contact_svc):
+    contact_svc.report = {
+        'HTTP': [
+            {
+                "paw": "test_1",
+                "instructions": [],
+                "date": "2021-08-29 15:00:00"
+            }
+        ],
+        'HTML': [
+            {
+                "paw": "test_2",
+                "instructions": [],
+                "date": "2021-08-31 15:00:00"
+            }
+        ]
+    }
+    return contact_svc.report
+
+
+class TestContactsApi:
+    async def test_get_http_contact_report(self, api_v2_client, api_cookies, contact_report_data):
+        resp = await api_v2_client.get('/api/v2/contacts/HTTP', cookies=api_cookies)
```

Comment:
we should also allow users to submit lowercase and have the API manager recapitalize as necessary. And we can try both in our tests